.github/workflows/doc-publish.yml

```diff
@@ -0,0 +1,104 @@
+# Copyright (c) 2020 Linaro Limited.
+# SPDX-License-Identifier: Apache-2.0
+
+name: Doc build for Release or Daily
+
+# Either a daily based on schedule/cron or only on tag push
+on:
+  schedule:
+    - cron:  '50 22 * * *'
+  push:
+    tags:
+    - '*'
```

Comment:
should we look into the release trigger?

```
 release:
    types: # This configuration does not affect the page_build event above
      - created
```

https://help.github.com/en/actions/reference/events-that-trigger-workflows

Reply:
Since we do annotated tags by hand I felt that was more useful than release.  Also, I think this will get trigged if one does a GH release in the case of adding a tag via that means.